sdk/personalizer/azure-ai-personalizer/tests/test_configuration_async.py

```diff
@@ -0,0 +1,72 @@
+from devtools_testutils import AzureRecordedTestCase
+import helpers
+import asyncio
+
+
+def configuration_equals(actual, expected):
+    assert actual.get('modelExportFrequency') == expected.get('modelExportFrequency')
+    assert actual.get('defaultReward') == expected.get('defaultReward')
+    assert actual.get('modelRetrainDays') == expected.get('modelRetrainDays')
+    assert actual.get('rewardWaitTime') == expected.get('rewardWaitTime')
+    assert actual.get('explorationPercentage') == expected.get('explorationPercentage')
+    assert actual.get('logRetentionDays') == expected.get('logRetentionDays')
+
+
+def policy_equals(actual, expected):
+    assert actual.get('arguments') == expected.get('arguments')
+
+
+class TestConfigurationAsync(AzureRecordedTestCase):
+
+    @helpers.PersonalizerPreparer()
+    async def test_update_configuration(self, **kwargs):
```

Comment:
you might need to add a @pytest.mark.asyncio to these async tests. I don't remember if the old framework handles this for you or not

Reply:
Fixed.